actions/terraform-checks/action.yml

```diff
@@ -20,6 +42,12 @@ runs:
         cd infrastructure
         terraform fmt -check $(find . -name '*tf')
 
+    - name: "INFO: Checking Terraform Format of Files"
+      shell: bash
+      run: |
+        cd infrastructure
+        terraform fmt -check $(find . -name '*tf')
```

Comment:
Am I missing something? Isn't this the step from above repeated?

Reply:
Not sure how I managed this - removed repeat and kept the better name!